_includes/post__meta.html

```diff
@@ -0,0 +1,29 @@
+{% assign page = post | default: page %}
```

Comment:
I think stuff like `p="post"` can be dropped as well since you've already done this. There are 4 of them I can spot for now.

Reply:
oops... missed those out, it's fixed now, thanks!